config/secrets.yml

```diff
@@ -60,7 +60,9 @@ default: &default
 
   country_statistics_api:
     url: <%= ENV['COUNTRY_STATISTICS_API_URL'] %>
-    national_endpoint: <%= ENV['API_NATIONAL_ENDPOINT'] %>
+    kba_endpoint: <%= ENV['API_KBA_ENDPOINT'] %>
+    prot_endpoint: <%= ENV['API_PROTECTION_ENDPOINT'] %>
+    conn_endpoint: <%= ENV['API_CONNECTION_ENDPOINT'] %>
```

Comment:
@lucacug Could you please add these also in the env.example file?

Reply:
Thanks! Addressed